.eslintrc.json

```diff
@@ -0,0 +1,45 @@
+{
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+        "ecmaVersion": 2017,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "jsx": true,
```

Comment:
Possibly. It's part of my standard eslint config. I thought about removing
it, then thought better of it since we are planning a transition.